.taskcluster.yml

```diff
@@ -33,7 +33,7 @@ tasks:
           - >-
             git init repo &&
             cd repo &&
-            git fetch --depth 1 "$GIT_URL" "$GIT_REF" &&
+            git fetch --depth 25 "$GIT_URL" "$GIT_REF" &&
```

Comment:
Do we actually want to keep running tasks if the branch has already changed again after the push that this task is reacting to?

Reply:
I don't think I understand the question. Can you describe a situation that you're concerned about and how it could occur?

Reply:
Resolved at https://mozilla.logbot.info/servo/20190514#c16310425-c16310457